Shazzam/Shazzam/ViewModels/MainWindowViewModel.cs

```diff
@@ -70,9 +70,26 @@ private void ExploreCompiledShadersCommand_Execute()
             string path = Properties.Settings.Default.FolderPath_Output;
             if (System.IO.Directory.Exists(path))
             {
-                System.Diagnostics.Process.Start(path);
-            }
+                Process process = new()
```

Comment:
Process is an IDisposable interface, please add using, so the Close() could be removed with the finally block